todo_app/app.py

```diff
@@ -1,32 +1,57 @@
+from todo_app.trello_api_client import TrelloAPIClient
+from todo_app.view_model import ViewModel
 from flask import Flask, render_template, request, redirect
-from todo_app.data.session_items import get_items, add_item, delete_item, get_item, save_item
-from todo_app.flask_config import Config
-
-app = Flask(__name__)
-app.config.from_object(Config)
-
-@app.route('/')
-def index():
-    todo_items = sorted(get_items(), key=lambda items: items['status'], reverse = True)
-    return render_template('index.html', todo_items = todo_items)
-
-@app.route('/', methods=['POST'])
-def add_todo_item():
-    title = request.form.get('add-item')
-    add_item(title)
-    return redirect('/')
-
-@app.route('/complete/<id>', methods=['POST'])
-def complete(id):
-    item = get_item(id)
-    item['status'] = 'Completed'
-    save_item(item)
-    return redirect('/')
-
-@app.route('/delete/<id>', methods=['POST'])
-def delete(id):
-    delete_item(id)
-    return redirect('/')
+
+trello_api_client = TrelloAPIClient()
+boards = trello_api_client.get_boards()
```

Comment:
This line runs before your .env.test loading and mocking have had a chance to take effect in your integration tests so your integration tests talk to real Trello.

Fortunately I don't think it's used anyway, so you can just get rid of it.